app/products/playbooks/actions/remote/runs.ts

```diff
@@ -136,6 +136,20 @@ export const setOwner = async (serverUrl: string, playbookRunId: string, ownerId
     }
 };
 
+export const renamePlaybookRun = async (serverUrl: string, playbookRunId: string, newName: string) => {
+    try {
+        const client = NetworkManager.getClient(serverUrl);
+        await client.patchPlaybookRun(playbookRunId, {name: newName});
+
+        await localRenamePlaybookRun(serverUrl, playbookRunId, newName);
```

Comment:
Again, ignoring potential errors. And probably assuming in the tests that `localRenamePlaybookRun` throws, when it doesn't.